include/DDpackage.h

```diff
@@ -218,8 +219,12 @@ namespace dd {
         unsigned long      UTcol{}, UTmatch{}, UTlookups{}; // counter for collisions / matches in hash tables
         ComplexNumbers     cn;                              // instance of the complex number handler
 
-        Package():
-            cn(ComplexNumbers()){};
+        Package(unsigned short nqubits = 128):
```

Comment:
shouldn't this member be call something like `maxQubits` or `maxN`?
It does not really capture the number of qubits, does it?

Reply:
This constructor should be `explicit`.